src/appkit.cs

```diff
@@ -15099,6 +15172,15 @@ interface NSStoryboard {
 		[Static]
 		[NullAllowed, Export ("mainStoryboard", ArgumentSemantic.Strong)]
 		NSStoryboard MainStoryboard { get; }
+
+		[Mac (10,15)]
+		[Export ("instantiateInitialControllerWithCreator:")]
+		[return: NullAllowed]
+		NSObject InstantiateInitialController ([NullAllowed] NSStoryboardControllerCreator block);
```

Comment:
docs says it returns `id`
but I suspect it returns a (subclass of) `NSViewController`
double check how iOS `UIStoryboard` does its binding